### ISSUE TITLE
Mac OS Ventura only shows/adds entry in Accessibility on first left-click on AutoRaise's icon after install

### ISSUE THREAD

**Issue by claudiusraphael** [sbmpost/AutoRaise#148]
It should be known by now, but somehow magically nobody even dares to mention that from Ventura on upwards the behavior for Accessibility-entries has changed, namely there simply will be no entry if the app itself does not request an entry. AutoRaise indeed requests to add an entry to Accessibility, but this is only triggered after installing and launching AutoRaise and then on the first click with the main (regularly the left) button.

In the README.MD there is no mention of that. The problem is that you can right-click (as mentioned in the steps) how often you like, it still will not enable/request the entry in Accessibility - this solely happens on the first left-click on the icon of AutoRaise.

Please, consider adding "left-clicking on AutoRaise to be able to enable it in Accessibility on Ventura and later versions of Mac OS" BEFORE the step to right-click to open preferences.

Aside of that, thanks for creating AutoRaise, maintaining it and sharing it freely - much appreciated, as it was one of those things i never was able to wrap my head around how people are able to use this system efficiently without auto-focus (i don't care about raise as i do use stacking layout exclusively for those few apps/tools that can actually stay on top - lol, while you are at it you do not by any chance also created a pin-window/always-on-top for Mac OS?)

Thanks!

**Comment by sbmpost** [comment]
@claudiusraphael 

I suppose step 6 in the README can be changed from:

**Comment by claudiusraphael** [comment]
Your textual change sounds good to me. Just tested today on 14 beta and its the same behavior as in Ventura, so will be helpful to users to mention it the way you proposed (Click on the balloon ..).

Also thanks for the insight what window behavior types are native to macos. Would that imply that only modal windows are allowed to always stay on top or are there alternatives that (from the users viewpoint) have a similar effect?

**Comment by sbmpost** [comment]
@claudiusraphael 

> Would that imply that only modal windows are allowed to always stay on top or are there alternatives that (from the users viewpoint) have a similar effect?

I think as an application developer, you can specify something like a "stay on top" attribute for your window. This attribute is only accessible from within the application I presume.

P.S doing a search on the Internet pointed me to "bettertouchtool" which apparently is able to pin a window. You might give it a try. I know of other users using AutoRaise in combination with that tool.
